libs/etterlatte-kafka/src/main/kotlin/kafka/Kafkakonfigurasjon.kt

```diff
@@ -46,6 +46,7 @@ abstract class Kafkakonfigurasjon<T>(
             put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, deserializerClass)
 
             put(Avrokonstanter.BASIC_AUTH_CREDENTIALS_SOURCE, "USER_INFO")
+            put("KAFKA_CONSUMER_GROUP_ID", env["NAIS_APP_NAME"]!!.replace("-", ""))
```

Comment:
Har vi manglet dette?

Reply:
Ja, vi har per no satt det eksplisitt for r&r-appane, men ikkje for dei som lyttar på andre kafka-topics (https://nav-it.slack.com/archives/C01PG1JM9D4/p1706082992031449)

Reply:
:O